package.json

```diff
@@ -24,7 +24,7 @@
     "@primer/css": "^14.4.0",
     "@snapshot-labs/lock": "github:snapshot-labs/lock#master",
     "@snapshot-labs/snapshot-spaces": "github:snapshot-labs/snapshot-spaces#master",
-    "@snapshot-labs/snapshot.js": "github:snapshot-labs/snapshot.js#master",
+    "@snapshot-labs/snapshot.js": "file:../snapshot.js",
```

Comment:
Hi, can you revert this change otherwise in will not build a preview.

Reply:
hello @samuveth thank you for taking a look at my code, yeah that is actually for the development, because it is make it easier for me to develop in local, because that package is referring to the list of plugin in snapshot.js repo, so i also make a fork of snapshot.js and also make a draft pull request,im gonna make it a pull request now so you can maybe take a look and merge it if it's good, then after that i will also change the package.json like before here is the pull request im talking about to show the commentBox as a plugin https://github.com/snapshot-labs/snapshot.js/pull/358

Reply:
You can keep these changes locally, no need to push them.

Reply:
> You can keep these changes locally, no need to push them.

can i change this after im done with the development? because my plugin still not finish anyway 😅 , so maybe i will change it later after im done with the development, if its ok with you? and that's why it's still draft as well coz im not ready to be reviewed yet coz still not done yet

Reply:
ok done i resolve the conflict